zubhub_frontend/zubhub/src/views/activity_details/MakeActivity.js

```diff
@@ -0,0 +1,126 @@
+import ReactDOM from 'react-dom';
+import clsx from 'clsx';
+import ReactQuill from 'react-quill';
+import QRCode from 'qrcode.react';
+import { images } from '../../assets/images';
+import { Collapsible, CustomButton, Gallery, Modal, Pill } from '../../components';
+import { Container, Typography, Box, makeStyles, Avatar } from '@material-ui/core';
+import styles from '../../assets/js/styles/index';
+import { activityDefailsStyles } from './ActivityDetails.styles';
+// import activity from '../../components/activity/activity';
+import SocialButtons from '../../components/social_share_buttons/socialShareButtons';
+export const MakeActivity = ({ activity, creator }) => {
+  const classes = makeStyles(activityDefailsStyles)();
+  const commonClasses = makeStyles(styles)();
+  return (
+    <>
+      <Container style={{ backgroundColor: 'white', border: 'solid 1.5rem white'}} className={classes.pdfStyle}>
+        <Container className={classes.pdfMainLogo}>
+          <Avatar className={classes.pdfLogoStyle} src={images.logo} alt={'Faridah_ux'} />
+        </Container>
+        <Box className={classes.verticalSpace}></Box>
+        <Container className={classes.justifyPdf}>
+          <Typography className={classes.pdfTitle}>{activity?.title}</Typography>
+          <Container className={`${classes.creatorProfilePdfStyle}`}>
+            <Avatar className={classes.creatorAvatarStyle} src={creator?.avatar} alt={'Faridah_ux'} />
+            <Typography>{activity?.creators[0].username}</Typography>
+          </Container>
+        </Container>
+        <Box className={classes.verticalSpace}></Box>
+        <Typography className={`${classes.pdfSubtitle}`}>Introduction</Typography>
+        <Typography className={classes.textStyle}>
+          {activity.introduction ? <div dangerouslySetInnerHTML={{ __html: activity.introduction }} /> : ''}
+        </Typography>
+        {activity.images?.length > 0 && (
+          <div className={classes.imageStyle}>
+            {activity.images.map((img, index) => (
+              <img
+                key={index}
+                src={img.image?.file_url}
+                alt={`Image ${index}`}
```

Comment:
this alt text is  not really helpful to anyone who doesn't speak english. we should translate this. Also maybe improve the alt text? `image 0` is not exactly a very descriptive alt